src/assets.rs

```diff
@@ -26,8 +32,9 @@ pub struct Asset {
     /// Optional asset icon.
     pub icon: Option<Icon>,
     mint_info: Option<MintInfo>,
+    // Skip this field when serializing to ensure it's `false` upon deserialization.
```

Comment:
This needs more justification. You can probably just copy our discussion from the comments on the last PR and clean it up a bit.